### ISSUE TITLE
Boss level - Coded with no power (06/14)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#111]
## Motivation/Problem Statement 💭❓:
Learn Python.

---

## Today's Goal/Solution 🥅:
I have been without power for the past 7 hrs. However, I am determined to get some coding done and maintain my streak. Tethered to my phone right now. I am going to download some exercises from Exercism so I can work locally.

---

## Result 🍓🍌🍉:
I completed three exercises and worked on a 4th.

---

## Observations & Next Steps 🔭👣:
I got some good work in. I have a couple of things to look up, but overall I'm happy that I was able to code with no power. I've downloaded several exercises onto my computer, so if this ever happens again, which I hope it won't, I'll have work that I can do locally.

---

## References 🔗:
Link any useful resources you may have found throughout the day.

---

## Extra/Fun (Optional) 🎈🎉🥳:
I started a new book. Read a few pages by flashlight. 😂

**Comment by r002** [comment]
Oh wow-- without power for seven hours and you still submitted a card for today!  😮  That is epic!!  You're a legend!!  👏👏👏

I hope everything is okay though!  Did the electricity go out in your neighborhood?  Stay safe!!

**Comment by anitabe404** [comment]
@r002 There was a thunderstorm in my area, and we lost power around 3 pm. It impacted 7 houses in our neighborhood, so I felt salty when I went outside and saw lights on a few houses down. We just got it back about a half-hour ago. The first thing I started brainstorming was how to submit my GitHub ticket without power, rofl. I'm terrible. 😂😭 I just really did not want to lose my streak.

We were safe. Fortunately, the house stayed cool, and I had enough power to complete what I needed. Dinner by flashlight and such.

**Comment by r002** [comment]
> The first thing I started brainstorming was how to submit my GitHub ticket without power, rofl. I'm terrible. 😂😭

No, you are **_awesome!!_**  I think that's immensely inspiring that was your first thought!  I'm the same way-- for us `Type A` people, the drive to "get the gold star" for the day runs _**strong.**_  Gotta keep the streak alive!

I'm glad you guys are okay though!  Dinner by flashlight... haha.  There are times when I, too, have eaten dinner to only the faint, liquid crystal fluorescence of my LED displays... 😅  Back in my bank days, we commonly called the practice, "feeding the monitor."  😎

**Comment by anitabe404** [comment]
Sometimes you have to do what you have to do. LOL
